edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/CloudProxy.cs

```diff
@@ -155,6 +155,7 @@ public async Task SendMessageBatchAsync(IEnumerable<IMessage> inputMessages)
         {
             IMessageConverter<Message> converter = this.messageConverterProvider.Get<Message>();
             string metricOutputRoute = null;
+            uint priority = uint.MaxValue;
```

Comment:
should this be `RouteFactory.DefaultPriority`?

Reply:
I'm trying to signify an invalid value here, as opposed to the default value.  If there are no inputMessages, DefaultPriority has a good chance of being misleading.

---
In reply to: [397345757](https://github.com/Azure/iotedge/pull/2730#discussion_r397345757) [](ancestors = 397345757)